spec/features/cases/offender_sar/case_creating_spec.rb

```diff
@@ -249,7 +265,7 @@ def expect_to_have_correct_sending_address
     expect(cases_show_page).to have_content "Address\n22 High Street"
   end
 
-  def and_back_previous_step_to_change_to_non_third_party
```

Comment:
This method is used elsewhere, so you'll need to make the name change there as well.